parser.y

```diff
@@ -2709,6 +2709,10 @@ EnforcedOrNot:
 	{
 		$$ = false
 	}
+|	not2 "ENFORCED"
```

Comment:
I prefer introducing a 

```yaml
NotSym:
    not
|   not2
```

and replace all mentions of `"NOT"` by `NotSym` (similar to `ValueSym`).